src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -6779,11 +6778,11 @@ private static void loopChecks2(List<MethodHandle> step, List<MethodHandle> pred
             int pc = h.type().parameterCount();
             int tpsize = targetParams.size();
             return pc < tpsize ? dropArguments0(h, pc, targetParams.subList(pc, tpsize)) : h;
-        }).collect(Collectors.toList());
+        }).toList();
     }
 
     private static List<MethodHandle> fixArities(List<MethodHandle> hs) {
-        return hs.stream().map(MethodHandle::asFixedArity).collect(Collectors.toList());
+        return hs.stream().map(MethodHandle::asFixedArity).toList();
     }
 
     /**
```

Comment:
I think the import of Collectors can be dropped in this file? And maybe a few other files too?

Reply:
The import can be dropped from the jdk.internal.module.* classes too.

jdk.internal.module.IllegalAccessLogger will likely be removed as part of JEP 403.